python/ray/tests/test_node_manager.py

```diff
@@ -238,12 +238,16 @@ def test_reference_global_import_does_not_leak_worker_upon_driver_exit(call_ray_
 import numpy as np
 import tensorflow
 
+@ray.remote
```

Comment:
can you also add "@ray.remote(max_retries=0)", to maintain the same behavior that no retry?